client/src/FormFrontend/FormFrontend.js

```diff
@@ -167,8 +167,10 @@ class FormFrontend extends React.Component {
 
     const user_fields = _.omitBy(
       template,
-      ({ form_type }, key) => key === "meta" || !form_type
+      // keeping key === "faq" until faq content is made
+      ({ form_type }, key) => key === "meta" || key === "faq" || !form_type
```

Comment:
How about a `hidden` property in the template (I'd put it at the top, above required), not showing any fields with `hidden: true`? That way the templates control this and only the form backend needs to redeploy to change the fields.

Bonus, it makes "meta" less of a special case field. The client needs to know less about it, it's just another `hidden` field when it comes to field rendering.